diff --git a/Source/ocdm/CMakeLists.txt b/Source/ocdm/CMakeLists.txt
index f7e7ad1..7553de4 100644
--- a/Source/ocdm/CMakeLists.txt
+++ b/Source/ocdm/CMakeLists.txt
@@ -19,11 +19,11 @@ cmake_minimum_required(VERSION 3.15)

 find_package(Thunder)

-project(OCDM)
+project(ocdm)

 project_version(1.0.0)

-set(TARGET Client${PROJECT_NAME})
+set(TARGET ${PROJECT_NAME})

 message("Setup ${TARGET} v${PROJECT_VERSION}")
